models/fact.js

```diff
@@ -71,23 +60,28 @@ function getDefaultMsg(number, type, options) {
       "nothing remarkable happened",
       "the Earth probably went around the Sun",
       "nothing interesting came to pass",
-      "we do not know what happened"
-    ]
+      "we do not know what happened",
+    ],
   }[type];
 
   var data = {
-    text: utils.randomChoice(defaultMsgs)
+    text: utils.randomChoice(defaultMsgs),
   };
 
-  return getSentence(options.fragment, number, type, data);
+  return getSentence({
+    wantFragment: options.fragment,
```

Comment:
yea, in this case i think the value of `wantFragment` will be a boolean or undefined, not an object. i could be wrong though. in any event, i *think* this means you should just be able to set `wantFragment: true` in those earlier tests.